tests/support/flattened/utility.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+// module Main
+
+exports.same = function(a) {
```

Comment:
Agreed. It's testing reference equality, so not a property we'd usually want to care about. There is `refEq` in the prelude for the `Prim` types, but it's not exported.